rows.go

```diff
@@ -31,49 +31,71 @@ type Row struct {
 
 // Cell represents the data in a single cell as a consumable format.
 type Cell struct {
-	Column string // E.G   A, B, C
-	Row    int
-	Value  string
+	Column      string // E.G   A, B, C
+	ColumnIndex int
+	Row         int
+	Value       string
+	Type        CellType
 }
 
+type CellType int
+
+const (
+	TypeNil CellType = iota
+	TypeString
+	TypeNumerical
+	TypeDateTime
+	TypeDate
+	TypeBoolean
```

Comment:
I think these would make some more sense as string constants, rather than using iota. 

I know it's a kinda defacto for doing a 'enum' style interface, but it doesn't make any sense if the data ever gets serialised. - having 
`const TypeNil CellType = "TYPE_NIL"` at least serialises to a string.

Reply:
Is it actually possible to get a `NIL` cell type, or is this just from returning errors?

Reply:
Exposing a lot of types feeling a bit yucky - it's going to make any future changes difficult to not break backwards compatibility. Wonder if there's a nicer way.